src/main/java/com/Saurabh/GitHubAiAPI/StringManipulation.java

```diff
@@ -45,4 +49,11 @@ public static String findVowels(String input) {
         }
         return vowels.toString();
     }
-}
+
+    // Helper method to check if a string is a palindrome
+    public static boolean isPalindrome(String input) {
```

Comment:
The isPalindrome method could be optimized by checking for palindrome in-place without creating a reversed string. This would reduce the space complexity from O(n) to O(1).